lib/interface.ts

```diff
@@ -21,6 +23,10 @@ export interface IOurResponse extends restify.Response {
     sendEnd?: (status: any, message: string) => void;
 }
 
+export interface IOurRequestHandler {
```

Comment:
I don't see a specific need for this interface here. The only usage is in `request-handler.ts` as type alias , which I think we can put this declaration there.